OrchardCore.Commerce/Views/AddressField_Edit.cshtml

```diff
@@ -4,74 +4,74 @@
 
 @{ var settings = Model.PartFieldDefinition.GetSettings<AddressPartFieldSettings>(); }
 <div class="row">
-    <div class="form-group col-md-12">
+    <div class="mb-3 col-md-12">
```

Comment:
What's the reason for removing `form-group` classes and adding margin utility classes instead?

Reply:
form-group has been removed from BS5 and it was basically only adding a bottom margin.